pkgs/development/python-modules/pymssql/default.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  freetds,
+  krb5,
+  openssl,
+  cython,
+  gevent,
+  psutil,
+  pytestCheckHook,
+  setuptools-scm,
+  sqlalchemy,
+  tomli,
+}:
+
+buildPythonPackage rec {
+  pname = "pymssql";
+  version = "2.3.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-8DTka1aAYdFxSPHe3qZI+dX2tzQOjP/g7bqhcTy0q6s=";
+  };
+
+  build-system = [
+    cython
+    setuptools-scm
+    tomli
+  ];
+
+  buildInputs = [
+    freetds
+    krb5
```

Comment:
This is supposed to be the C library? See https://github.com/NixOS/nixpkgs/pull/333058.

Reply:
Yes. I'll also explicitly inherit krb5 from pkgs

Reply:
Please call it `krb5-c` as in that PR.